### ISSUE TITLE
ODS exists with no settings.xml

### ISSUE THREAD

**Issue by Tux** [Corion/Spreadsheet-ReadSXC#2]
I got [this ODS](https://www.rvig.nl/binaries/rvig/documenten/richtlijnen/2018/09/20/test-burgerservicenummers-bsn-en-a-nummers-inclusief-omnummertabel/20200617_Testnummers_inclusief_omnummertabel_GBA-V.ods)  form a [government site](https://www.rvig.nl/documenten/richtlijnen/2018/09/20/test-burgerservicenummers-bsn-en-a-nummers-inclusief-omnummertabel), and it does not parse at all as it does not contain a settings.xml
```
$ sr -d 20200617_Testnummers_inclusief_omnummertabel_GBA-V.ods
ODS parser cannot parse data: Want to read settings.xml' but it doesn't exist! at /home/merijn/bin/sr line 28.
Exit 255
$ unzip -l 20200617_Testnummers_inclusief_omnummertabel_GBA-V.ods
Archive:  20200617_Testnummers_inclusief_omnummertabel_GBA-V.ods
  Length      Date    Time    Name
---------  ---------- -----   ----
       46  1980-01-01 00:00   mimetype
     4616  1980-01-01 00:00   styles.xml
  1923445  1980-01-01 00:00   content.xml
      532  1980-01-01 00:00   META-INF/manifest.xml
      772  1980-01-01 00:00   meta.xml
---------                     -------
  1929411                     5 files
$ file 20200617_Testnummers_inclusief_omnummertabel_GBA-V.ods
20200617_Testnummers_inclusief_omnummertabel_GBA-V.ods: OpenDocument Spreadsheet
$ perl -MSpreadsheet::ParseODS -wE'my $p = Spreadsheet::ParseODS->new;my $wb = $p->parse (glob ("20*.ods"))'
Want to read settings.xml' but it doesn't exist! at -e line 1.
```
If I unzip, I see in meta.xml
``` xml
  <office:meta>
    <meta:generator>MicrosoftOffice/16.0 MicrosoftExcel/CalculationVersion-14420</meta:generator>
```
And the (formatted) META-INF/manifest.xml shows
``` xml
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<manifest:manifest xmlns:manifest="urn:oasis:names:tc:opendocument:xmlns:manifest:1.0">
  <manifest:file-entry manifest:full-path="/" manifest:media-type="application/vnd.oasis.opendocument.spreadsheet"/>
  <manifest:file-entry manifest:full-path="styles.xml" manifest:media-type="text/xml"/>
  <manifest:file-entry manifest:full-path="content.xml" manifest:media-type="text/xml"/>
  <manifest:file-entry manifest:full-path="meta.xml" manifest:media-type="text/xml"/>
  </manifest:manifest>
```

**Comment by shawnlaffan** [comment]
It would seem the MS Office removes the settings.xml file (and perhaps other info) when saving to the ods format.

Verified using an ods file exported from Google Sheets.

**Comment by Corion** [comment]
I'll add a test case to address this issue!

**Comment by Corion** [comment]
This should be addressed by 0ddf4fb1e458cd3a9a5160abfd4d88f2ba6c4b9b , on its way to CPAN as 0.26
